concepts/psp-replacement/README.md

```diff
@@ -0,0 +1,31 @@
+| Privilege Escalation     | No opinion                                                                                                                                                                                                                                                              | Privilege escalation is not allowed<br><br>`spec.<anycontainer>[*].securityContext.allowPrivilegeEscalation = false`                                                                                                                         |
+| Running as Non-root      | No opinion                                                                                                                                                                                                                                                              | Containers must be required to run as non-root users<br><br>`spec.<anycontainer>[*].securityContext.runAsNonRoot = true`                                                                                                                     |
+| Running as Non-root user | No Opinion<br><br>If this setting is needed, it should never been set to 0 because this is the ID of the root user.                                                                                                                                                     | Containers must set runAsUser to a value different from 0, optimal are values > 10000 to limit frictions with existing user accounts<br><br>`spec.securityContext.runAsUser != 0`<br>`spec.<anycontainer>[*].securityContext.runAsUser != 0` |
+| Seccomp                  | Optional<br><br>Allowed values are:<br>- RuntimeDefault<br>- Localhost                                                                                                                                                                                                  | Mandatory<br><br>Allowed values are:<br>- RuntimeDefault<br>- Localhost                                                                                                                                                                      |
+| Capabilities             | Only additional capabilities from the list below are allowed.<br><br>- Undefined/nil<br>- AUDIT_WRITE<br>- CHOWN<br>- DAC_OVERRIDE<br>- FOWNER<br>- FSETID<br>- KILL<br>- MKNOD<br>- NET_BIND_SERVICE<br>- SETFCAP<br>- SETGID<br>- SETPCAP<br>- SETUID<br>- SYS_CHROOT | Containers must drop ALL capabilities, and are only permitted to add back the NET_BIND_SERVICE capability.                                                                                                                                   |
```

Comment:
I change the value of the 'Running as Non-root user' setting a bit but I didn't follow your suggestion because the developer can set it to 0 and there might be some circumstances where this is necessary.

Reply:
I didn't intend to change the meaning here, I simply applied the content guidelines. For the sake of language simplicity, we are not supposed to use passive voice forms. "Should" is also to be avoided in favor of "must" or "can". Hence the change from "should never" to "cannot". If you insist on keeping the original sentence, just change the form of "be" in "it should never been set" (from "been" to "be"), please.